pkgs/development/tools/misc/ttags/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib , fetchFromGitHub , rustPlatform }:
+
+rustPlatform.buildRustPackage {
+  pname = "ttags";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "npezza93";
+    repo = "ttags";
+    rev = "b8ced4cc71bbf27f514f0084b58e1808a0ea4164";
+    hash = "sha256-nWtsON6VEgXlBao7jxUMIvGJ/8k6cw1I/Zd+aSYzV1I=";
+  };
+
+
```

Comment:
this file could probably be run through nixfmt, and it'll clean up this double newline and a few other things in this file